{"name": "test_calc_sum_1[3-5-9]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 8 == 9", "trace": "input1 = 3, input2 = 5, output = 9\n\n    @pytest.mark.parametrize(\"input1,input2,output\",\n                            [\n                                (2,3,5),\n                                (3,3,6),\n                                (3,5,9)\n                            ]\n                             )\n    def test_calc_sum_1(input1,input2,output):\n        result = input1 + input2\n>       assert result == output\nE       assert 8 == 9\n\ntest_parametrization_demo.py:17: AssertionError"}, "parameters": [{"name": "input1", "value": "3"}, {"name": "input2", "value": "5"}, {"name": "output", "value": "9"}], "start": 1564755064066, "stop": 1564755064067, "uuid": "ece19966-81db-465d-8bd9-d66611e3663b", "historyId": "67f39ead40825024ec036b52e4f77686", "fullName": "test_parametrization_demo#test_calc_sum_1[3-5-9]", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_parametrization_demo"}, {"name": "subSuite"}, {"name": "host", "value": "lfbn-1-12791-90.w90-90.abo.wanadoo.fr"}, {"name": "thread", "value": "1734-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_parametrization_demo"}]}